ovirt-imageio.spec.in

```diff
@@ -224,6 +224,8 @@ Common resources used by oVirt imageio server and client
 %{python3_sitearch}/%{srcname}-*.egg-info
 %exclude %{python3_sitearch}/%{srcname}/client
 %exclude %{python3_sitearch}/%{srcname}/admin
+%define python3_pkgversion 3
+%define __python3 /usr/bin/python3
```

Comment:
Aren't these overwriting the defines above in the same condition block?

Reply:
Those macros were needed only for `%{python3_sitearch}` usage above, so we change them back to the default value at the end. This should fix the issue with the order of package creation, which it not based on the position in specfile (see more details in OST chat).